docker/hail-ubuntu/Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM {{ global.docker_prefix }}/ubuntu:jammy-20230624
+ARG DOCKER_PREFIX={{ global.docker_prefix }}
```

Comment:
This default is for backward compatibility with the current CI that doesn't know about `ARG` and just wants to render this file template. We could presumably replace this with `ARG DOCKER_PREFIX=docker.io` so someone can build this image without needing to set up a whole registry / copy over images / set up k8s secrets to get the `global-config`